app/views/layouts/_messages.html.erb

```diff
@@ -1,12 +1,14 @@
 <% unless flash.empty? %>
-  <% flash.each do |name, msg| %>
-    <% if msg.is_a?(String) %>
-      <% if name == 'notice' then name = 'info' end %>
-      <div class="usa-alert usa-alert__paragraph usa-alert--<%= name %>" >
-        <div class="usa-alert__body">
-          <p class="usa-alert__text"><%= msg %></p>
+  <div class="margin-bottom-2">
+    <% flash.each do |name, msg| %>
+      <% if msg.is_a?(String) %>
+        <% if name == 'notice' then name = 'info' end %>
+        <div class="usa-alert usa-alert__paragraph usa-alert--<%= name %>" >
```

Comment:
`if ... then` is super uncommon, WDYT of adding a new variable and using a ternary?

```suggestion
        <% alert_name = name == 'notice' ? 'info' : name %>
        <div class="usa-alert usa-alert__paragraph usa-alert--<%= alert_name %>" >
```

Reply:
Just indented this partial to add some padding but I'm happy to update this.